.circleci/config.yml

```diff
@@ -9,6 +9,8 @@ jobs:
   build:
     docker:
       - image: circleci/python:3.6
+    resource_class: large
+    parallelism: 3
```

Comment:
@dippindots should we also apply this to the `validate_all_studies` job?

Reply:
sure, let me update this